### ISSUE TITLE
Svelte 3 Support

### ISSUE THREAD

**Issue by PaulMaly** [UnwrittenFun/svelte-vscode#35]
Hi! Thanks for this great tool! 

I see v 0.7.0 seems to support Svelte 3. So, maybe this is an issue:

```
A component can only have a default exportsvelte(named-export)
```

**Comment by Ragnoroct** [comment]
I'm having this same problem.

**Comment by jamesbirtles** [comment]
make sure you have svelte v3 installed locally in your project and it should pick it up

**Comment by zigomir** [comment]
Ah, cool. I had same issue, just needed to reload VSCode after `npm install`. Thanks!

**Comment by PaulMaly** [comment]
Still not working to me:
<img width="1063" alt="Снимок экрана 2019-04-30 в 9 52 25" src="https://user-images.githubusercontent.com/4378873/56944875-c7b8fb00-6b2d-11e9-9661-a6f43fc7b91d.png">


p/s  Reloaded VSCode twice

**Comment by alshdavid** [comment]
![image](https://user-images.githubusercontent.com/12656294/57010949-9b42d300-6c42-11e9-9f35-c8ffad1baff6.png)

![image](https://user-images.githubusercontent.com/12656294/57010955-a4cc3b00-6c42-11e9-885e-afe19d4777ec.png)

![image](https://user-images.githubusercontent.com/12656294/57010965-b6adde00-6c42-11e9-9f92-17221bf15433.png)

**Comment by PaulMaly** [comment]
@alshdavid Do you use svelte-ts-preprocess? I think it’s not supported by this plugin at all. And I’m not sure it’ll be. Maybe we should have additional plugin for TS support in Svelte.

**Comment by PaulMaly** [comment]
Sorry, issue not closed. Missed the button on mobile

**Comment by alshdavid** [comment]
That's okay, I'm sorry, I added no context!

Yeah that's using TypeScript preprocessor, unfortunate. I am looking forward to using TypeScript with Svelte.

**Comment by morleytatro** [comment]
I'm having the same problem. Irrespective of the number of times I reload, it's not picking up v3 syntax.  I'm specifying preprocess in a svelte.config.js file, but I'm not using svelte-ts-preprocess.

**Comment by jamesbirtles** [comment]
try `lang="typescript"` or `type="text/typescript"` as well as a svelte.config.js

Hopefully will be making this easier soon

**Comment by jamesbirtles** [comment]
original issue should have been resolved in 0.8.0, if there are other problems please open an issue specifically for that
